Styled Questions Button

kyelindholm opened the pull request:
Mostly just yoinked Kevin's button style to make the page more consistent looking. The background color/cursor also now changes when hovered to indicate clickability.

ks10825n reviewed (approved):
Everything is coming together!

kyelindholm commented:
Also fixed the bug where we have to click the More Answered Questions button twice before it actually renders. This was because setState is async, so the render function was getting called before state was actually set. SetState can also take a callback, so I just moved the render function call into a CB, which fixed it!
